app/models/plate_appearance.rb

```diff
@@ -18,194 +18,71 @@ class PlateAppearance < ActiveRecord::Base
   def play_out_at_bat
     determine_outcome
     code = outcome.code
+    code.slice!(" with injury") # Implement injuries later
+    code.slice!(" plus injury") # Implement injuries later
 
-    if code == "strikeout"
+    # TODO: Sort these by probability at some point
+    if simple_out?(code)
       inning.add_one_out
-    elsif code.downcase.first(3) == "fly"
-      #### FLYBALL A #######
-      if code =~ /A$/
-        if inning.two_outs
-          inning.retire_the_side
-        else
-          inning.add_one_out
-          inning.runners_advance_one_base
-        end
-      #### FLYBALL B #######
-      elsif code =~ /B\.?$/
-        if inning.two_outs
-          inning.retire_the_side
-        else
-          inning.add_one_out
-          inning.runner_from_third_scores
-        end
-      #### FLYBALL C #######
-      else
+    elsif simple_single_like_event?(code)
+      inning.runners_advance_one_base
+      inning.man_on_first = batter
+    elsif home_run?(code)
+      inning.everyone_scores_including_batter
+    elsif single_with_two_asterisk?(code)
+      inning.runners_advance_two_bases
+      inning.man_on_first = batter
+    elsif double?(code)
+      inning.runners_advance_two_bases
+      inning.man_on_second = batter
+    elsif triple?(code)
+      inning.everyone_scores_except_batter
+      inning.man_on_third = batter
+    elsif groundball_a?(code)
+      if !inning.at_least_one_runner_forced? || inning.two_outs
         inning.add_one_out
+      else
+        inning.add_two_outs 
+        inning.man_on_first = nil
+        inning.runners_advance_one_base unless inning.three_outs?
       end
-    elsif code.downcase.first(2) == "gb"
-      # TODO: Add Groundball B...handle Groundball X in some way for now
-      if code =~ /A\+?$/
-        if inning.two_outs
-          inning.retire_the_side
```

Comment:
Looks like we don't need `retire_the_side` anymore. Remove the method definition and its unit test